client/packages/invoices/src/InboundShipment/api/hooks/line/useNewSupplierReturnLines.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  useNotification,
+  useQuery,
+  useTableStore,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useInboundApi } from '../utils/useInboundApi';
+import { useInboundRows } from './useInboundRows';
+
+export const useNewSupplierReturnLines = () => {
```

Comment:
Have added this as an inbound shipment API hook... but it may be better located with other returns-related API code?